tests/update_reference_data.sh

```diff
@@ -19,7 +19,7 @@ copyToReferenceDir () {
 }
 
 tests=${@:2}
-test -z "$tests" && tests="spe11 spe12 spe3 spe9 norne_init msw_2d_h"
+test -z "$tests" && tests="spe11 spe12 spe12p spe3 spe9 norne_init msw_2d_h"
```

Comment:
the script allows the user to specify which test to update. not used on jenkins, but very handy locally (see later comments). it's just a unique string that should correlate to case name in opm-data somewhat (since it ends up in the commit message). unless specified, we check all tests so name has to be appended to this list.